doc/device-samples/iot-hub-prerequisites.md

```diff
@@ -0,0 +1,47 @@
+# 🔨 Create an IoT hub
```

Comment:
Can we just link to some existing documentation on creating an IoT Hub? IMO we shouldn't have multiple places documenting the same thing. There might also be other stuff in the `doc` directory that can also just be an MSDocs link.

Reply:
the existing IoT Hub docs are long and confusing. Thus the reason I wanted to create a shortened and targeted version of them. Something that gets to the point and is easy to follow. We can discuss.